packages/com.retroarch.yml

```diff
@@ -0,0 +1,283 @@
+title: Retroarch
+iconUri: https://github.com/webosbrew/RetroArch/blob/webos-tv-support/webos/icon160.png
+manifestUrl: https://github.com/webosbrew/RetroArch/releases/download/v1.9.1-webos.0/com.retroarch.manifest.json
```

Comment:
Could you change to `https://github.com/webosbrew/RetroArch/releases/latest/download/com.retroarch.manifest.json`?

Reply:
Changed.